docs/ATF/Table of Contents/index.md

```diff
@@ -1,4 +1,5 @@
-# Table of contents
+# ATF Software Architecture Document (SAD)
+## Table of contents
```

Comment:
@LuxoftAKutsan, other chapters is more clear due to context in them.
But you can check it with a link above.